pom.xml

```diff
@@ -23,7 +23,7 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/clover-plugin.git</connection>
+        <connection>scm:git:https://github.com/jenkinsci/clover-plugin.git</connection>
```

Comment:
I'd rather change to it SSH. 

In case of HTTPS it will ask for credentials, which is less convenient during the release process (mvn release:prepare release:perform).

Reply:
As far as I understand it, this is the "read-only" connection.  Previously it was read-only over an unecrypted transport.  With `https` it is read-only over an encrypted transport.  If it is changed to `ssh` that will be different than most other components in the Jenkins project.

Reply:
bq. this is the "read-only" connection.

Good point. We can have https:// here.